.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Feature request
+about: Suggest an idea NethServer project
```

Comment:
I'd change it to:
"Suggest and idea for NethServer project"

Reply:
Yep, that was a typo

Reply:
I'd say "Suggest an idea to the NethServer project".